src/app/content/highlights/hooks/locationChange.spec.ts

```diff
@@ -149,40 +149,63 @@ describe('locationChange', () => {
   });
 
   describe('error handling', () => {
-    it('doesn\'t show a toast if hook ran because of page\'s focus changing', async() => {
+    it('throws HighlightLoadError', async() => {
```

Comment:
the old test should not be removed

Reply:
I see that there some checks for receivePageFocus, but I have to look if they are correct

Reply:
But it doesn't `dispatch(addToast(...)` anymore. It's only about throwing the proper error.

Reply:
I think we should test if the hook with `receivePageFocus` will not throw `HighlightLoadError`